src/ezstream.c

```diff
@@ -158,6 +159,12 @@ buildReencodeCommand(const char *extension, const char *fileName,
 		xfree(dec_str);
 		dec_str = tmpStr;
 	}
+	if (strstr(dec_str, PLACEHOLDER_ALBUM) != NULL) {
+		char *tmpStr = replaceString(dec_str, PLACEHOLDER_ARTIST,
```

Comment:
My mistake, I thought I tested everything. I changed it to PLACEHOLDER_ALBUM, compiled, and test ed it.